lib/wallaby/browser.ex

```diff
@@ -783,9 +783,27 @@ defmodule Wallaby.Browser do
   end
 
   @doc """
-  Matches the Element's content with the provided text
+  Matches the Element's content with the provided text.
```

Comment:
@mhanberg Is using the word `Element` here ok (I see that it is used in other places) or would `Parent` be more correct here?

Reply:
"Matches the paren't content with the provided test." is probably okay.

Reply:
Updated to `parent's`.